charts/README.md

```diff
@@ -19,6 +19,19 @@
 helm repo add azuredisk-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/charts
 helm install azuredisk-csi-driver azuredisk-csi-driver/azuredisk-csi-driver --namespace kube-system
 ```
+## install Azure Disk CSI Driver V2 (Preview)
+For the case that your cluster's K8s version is below 1.21. Otherwise, please use the side-by-side installation
```

Comment:
The instructions are applicable to any K8s cluster version w/o the V1 driver installed. On AKS, however, the V1 driver is installed by default and the side-by-side instructions will need to be used here. You should indicate such here.